app/apollo/schema/subscription.js

```diff
@@ -112,12 +113,12 @@ const subscriptionSchema = gql`
      """
      Adds a subscription
      """
-     addSubscription(orgId: String! @sv, name: String! @sv, groups: [String!]! @sv, channelUuid: String! @sv, versionUuid: String! @sv): AddChannelSubscriptionReply!
```

Comment:
why was this `!` removed?

Reply:
It defaults to a blank array in the code if unset. Its so if someone wants to based the subscription off a clusterId, they dont need to pass the groups at all.